test/extensions/filters/http/cdn_loop/parser_fuzz_test.cc

```diff
@@ -0,0 +1,34 @@
+#include "common/common/statusor.h"
+
+#include "extensions/filters/http/cdn_loop/parser.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+#include "absl/strings/string_view.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+using Envoy::Extensions::HttpFilters::CdnLoop::Parser::parseCdnInfoList;
+using Envoy::Extensions::HttpFilters::CdnLoop::Parser::ParseContext;
+using Envoy::Extensions::HttpFilters::CdnLoop::Parser::ParsedCdnInfoList;
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  absl::string_view input(reinterpret_cast<const char*>(buf), len);
+  StatusOr<ParsedCdnInfoList> list = parseCdnInfoList(ParseContext(input));
+  if (list) {
+    // If we successfully parse input, we should make sure that cdn_ids we find appear in the input
+    // string in order.
+    size_t start = 0;
+    for (const absl::string_view cdn_id : list->cdnIds()) {
+      size_t pos = input.find(cdn_id, start);
+      EXPECT_NE(pos, absl::string_view::npos);
```

Comment:
very small nit: gtest assertions will work in the gtest environment, but don't trigger fatal crashes in the fuzzing setup. Could you change these to `ASSERT/FUZZ_ASSERT` please? (both will trigger fatal crashes, since fuzzers in oss-fuzz are always built with asserts)

Reply:
Done!